packages/ocean-lending/src/clients/oToken.ts

```diff
@@ -145,6 +116,11 @@ class OToken extends Client {
     return BigInt(supply);
   }
 
+  public async totalBorrows(): Promise<BigInt> {
+    const supply = await this.contract.methods.totalBorrows().call();
+    return BigInt(supply);
```

Comment:
I think just need to return as `string` type which is original type on chain

Reply:
or return number

Reply:
I think no need, just return it as pure as we can, 
the `Web3.utils.fromWei` is using param as string too

Reply:
if return `number`, FE needs to convert again to string which is unnecessary

Reply:
because in the integration tests, we are doing compares.

Reply:
not sure if FE will need to perform math operations. I think as number it is easier for those operations.

Reply:
Currently, FE doesn't have math operations, but if it had can easily convert to. Mostly using it for displaying